### PR TITLE
Fix/load

### LINKED ISSUE #183: Mint endpoint failes
**Describe the bug**
Mint api call returns a 500 error

**Not reproducible on alpha-goerli**

- [X] This issue is only present on Devnet and cannot be reproduced on alpha-goerli (check the box if true).

**To Reproduce**

1. Try to Mint 1 Eth with Bravos wallet; or make POST /mint request to devnet with Postman

**Expected behavior**
Get some Eth minted to the account.

**Screenshots**
Screenshot 2022-07-18 at 11 52 04

**Error Message in devnet logs***
```
[2022-07-18 11:43:21,765] ERROR in app: Exception on /mint [POST]
Traceback (most recent call last):
  File "/Users/claudio/.miniconda3/envs/starknet-dev-env/lib/python3.8/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/claudio/.miniconda3/envs/starknet-dev-env/lib/python3.8/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/claudio/.miniconda3/envs/starknet-dev-env/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/Users/claudio/.miniconda3/envs/starknet-dev-env/lib/python3.8/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/claudio/.miniconda3/envs/starknet-dev-env/lib/python3.8/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/Users/claudio/.miniconda3/envs/starknet-dev-env/lib/python3.8/site-packages/asgiref/sync.py", line 213, in __call__
    return call_result.result()
  File "/Users/claudio/.miniconda3/envs/starknet-dev-env/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/Users/claudio/.miniconda3/envs/starknet-dev-env/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/Users/claudio/.miniconda3/envs/starknet-dev-env/lib/python3.8/site-packages/asgiref/sync.py", line 279, in main_wrap
    result = await self.awaitable(*args, **kwargs)
  File "/Users/claudio/.miniconda3/envs/starknet-dev-env/lib/python3.8/site-packages/starknet_devnet/blueprints/base.py", line 126, in mint
    tx_hash_int = await FeeToken.mint(address, amount, state.starknet_wrapper)
  File "/Users/claudio/.miniconda3/envs/starknet-dev-env/lib/python3.8/site-packages/starknet_devnet/fee_token.py", line 101, in mint
    assert cls.contract
AssertionError
```

**Devnet version**

- I am using Devnet version: 0.2.5
- [ ] This happens with a dockerized Devnet (check the box if true).
- This does not appear on the following Devnet version:

**Machine specifications**

- OS: macOS Monterey 12.4.1
- CPU: Apple M1 Max
- Memory: 64GB

### PR DESCRIPTION
**Pull request by FabijanC**
## Usage related changes

- Fix #183 

## Development related changes

- Create instance of `FeeToken` in `StarknetWrapper`:
  - Stop using `FeeToken` as a class with class methods.
- Introduce two new test cases:
  - `test_mint_after_load`
  - `test_load_via_http_if_no_file`

## Checklist:

- [x] No linter errors
- [x] Performed a self-review of the code
- [x] Rebased to the last commit of the target branch (or merged it into my branch)
- [x] Documented the changes
- [x] Linked the issues which this PR resolves
- [x] Updated the tests
- [x] All tests are passing

### CODE REVIEW

File: starknet_devnet/fee_token.py [hunk 1]
```diff
@@ -63,18 +62,16 @@ async def deploy(cls, starknet: Starknet):
             }
         )
 
-        cls.contract = StarknetContract(
+        self.contract = StarknetContract(
```
Comment (tabaktoni) [inline starknet_devnet/fee_token.py:65]: Is it possible to unify usage of self and FeeToken

File: starknet_devnet/fee_token.py [hunk 2]
```diff
@@ -83,28 +80,20 @@ async def get_balance(cls, address: int) -> int:
...
             ],
             "signature": [],
             "contract_address": hex(cls.ADDRESS)
         }
-        transaction = InvokeFunction.load(transaction_data)
-        _, tx_hash, _ = await starknet_wrapper.invoke(transaction)
-        return tx_hash
+        return InvokeFunction.load(transaction_data)
+
+    async def mint_lite(self, to_address: int, amount: Uint256) -> None:
```
Comment (tabaktoni) [inline starknet_devnet/fee_token.py:97]: Can we add mint_lite also to starknet_wrapper so that similar functions are nearby?

**Review by tabaktoni — CHANGES_REQUESTED** [review]
I am not too happy about this solution, it will work but maybe we can think of some more elegant solution for the future.

**Review by tabaktoni — APPROVED** [review]

File: starknet_devnet/fee_token.py (hunk 2 above)
Comment (FabijanC) [reply starknet_devnet/fee_token.py]: https://github.com/Shard-Labs/starknet-devnet/pull/202/commits/8691e6aae1d3d329e28b8fc7917d25944f7246fb

**Pull request merged**